imjoy/imjoyPluginEngine.py

```diff
@@ -683,50 +697,84 @@ def connect(sid, environ):
 @sio.on("init_plugin", namespace=NAME_SPACE)
 async def on_init_plugin(sid, kwargs):
     try:
-        if sid in registered_sessions:
+        logger.info("init plugin %s", kwargs)
+        pid = kwargs["id"]
+        role = kwargs.get("role", "admin")
+        publish_id = kwargs.get("publish_id", None)
```

Comment:
```suggestion
        publish_id = kwargs.get("publish_id")
```
`None` is the default value returned if the key is missing in the dict.

Reply:
I see.